analyzer/codechecker_analyzer/cmd/fixit.py

```diff
@@ -152,7 +156,21 @@ def check_existence_and_collect(file_path):
             return False
         return True
 
+    def check_modification_and_collect(file_path):
+        """
+        Return True if the file was not modified after the given timestamp
+        otherwise collect it to a set named modified_files.
+        """
+        if util.get_last_mod_time(file_path) > modification_time:
```

Comment:
`get_last_mod_time` function may return with `None` in case of `OSError` exception. In this case you will compare None with a Number which will throw an exception in Python3. Similar to this: #2800